config/toml.go

```diff
@@ -83,6 +83,19 @@ proxy_app = "{{ .BaseConfig.ProxyApp }}"
 # A custom human readable name for this node
 moniker = "{{ .BaseConfig.Moniker }}"
 
+# Mode of Node: fullnode | validator | seednode (default: "fullnode")
+# Default value is fullnode, so you need to set it to "validator" if you want to run the node as validator
+# * fullnode (default)
+#   - all reactors
+#   - No priv_validator_key.json, priv_validator_state.json
+# * validator
+#   - all reactors
+#   - with priv_validator_key.json, priv_validator_state.json
+# * seednode
```

Comment:
How does this setting differ from `seed_mode` in the p2p section: https://github.com/tendermint/tendermint/blob/43c3e4265b5280b59e0d0af5f3f02e3141989b06/config/toml.go#L292-L296? Should this be removed?

Reply:
`seed_mode` is the setting for the functional part of the seed role, and `seednode` is the setting to lighten the node without using unnecessary reactors(e.g. consensus,) for acting as only seed independent of the seed function. 

Therefore, I think it would be better to keep the seed_mode setting because we may need a node that performs the seed function in `fullnode`. 

However, I think it would be better to change the name because it can confuse the word of `seed_mode` and the option of `seednode`.